docs/2.7.0/docs/daml-finance/tutorials/instrument-extension-packages/bond-extension.rst

```diff
@@ -271,3 +271,20 @@ How do I redeem a bond?
 
 On the redemption date, both the last coupon and the redemption amount will be paid. This is
 processed in the same way as a single coupon payment described above.
+
+How do I view the terms of a bond instrument?
+=============================================
+
+There are several ways to access the data of a contract:
+
+#. :ref:`fetch <daml-ref-fetch>` the contract using its contract ID. This requires the submitting
+   party to be a stakeholder of the contract.
+#. :ref:`fetchByKey <daml-ref-fetch-by-key>`: Similar to ``fetch``, but using a
+   :doc:`contract key </daml/reference/contract-keys>` instead. This requires authorization from at
+   least one stakeholder of the contract.
+#. ``GetView``: by calling this choice on the
```

Comment:
The `GetView` choice is a generic pattern we follow. I think it's worth explaining this in a generic section

Reply:
I was considering that as well. Maybe move the information here? 
https://docs.daml.com/2.7.0/daml-finance/tutorials/instrument-extension-packages/intro.html

Reply:
The `GetView` pattern is not instrument specific, so I would target a more generic location

Reply:
maybe a new `Patterns` page under the `Reference` section?

Reply:
Makes sense, we could add other patterns there as well.